pitest/src/main/java/org/pitest/coverage/execute/CoverageMinion.java

```diff
@@ -116,8 +115,8 @@ public static void main(final String[] args) {
 
   private static void enablePowerMockSupport() {
     // Bwahahahahahahaha
-    HotSwapAgent.addTransformer(new BendJavassistToMyWillTransformer(Prelude
-        .or(new Glob("javassist/*")),
+    HotSwapAgent.addTransformer(new BendJavassistToMyWillTransformer(
+        new Glob("javassist/*"),
```

Comment:
new Glob vs. toGlobPredicate() -- would rather use one approach either the factory method or the constructor but we shouldn't mix both. Maybe we should go with the factory method. What do you think?